src/datachain/lib/file.py

```diff
@@ -547,6 +571,17 @@ def save(self, destination: str, client_config: Optional[dict] = None):
 class ImageFile(File):
     """`DataModel` for reading image files."""
 
+    def get_info(self) -> "Image":
```

Comment:
let's rename it after all? I mean Image to ImageMeta or something? it is confusing tbh otherwise

Reply:
This is how it was initially, see [this comment](https://github.com/iterative/datachain/pull/814#pullrequestreview-2547435306). Me personally, me too prefer `ImageMeta` instead of `Image`, but we do have this conversation before and decided to do it this way.

cc @dmpetrov

Reply:
let's rename it please. Anything is better then just `Image` to my mind. It is the most confusing name for it.